webapp/components/posts_view_container.jsx

```diff
@@ -158,17 +157,6 @@ export default class PostsViewContainer extends React.Component {
             this.setState({scrollType: PostsView.SCROLL_TYPE_FREE});
         }
     }
-    shouldComponentUpdate(nextProps, nextState) {
```

Comment:
I guess that's fine, but I thought we were wanting to fix any cases where the stores were mutated in a way that would prevent this from working so that we could then remove the deep copying.